data/tlg0085/tlg003/__cts__.xml

```diff
@@ -1,8 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0085" xml:lang="grc" urn="urn:cts:greekLit:tlg0085.tlg003">
-  <ti:title xml:lang="lat">Prometheus Vinctus</ti:title>
   <ti:title xml:lang="eng">Prometheus Bound</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg003.opp-grc3" workUrn="urn:cts:greekLit:tlg0085.tlg003">
+  <ti:edition urn="urn:cts:greekLit:tlg0085.tlg003.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0085.tlg003">
     <ti:label xml:lang="grc">Προμηθευς δεσμωτης</ti:label>
```

Comment:
Προμηθεὺς Δεσμώτης

Reply:
That's funny I copied the title for the cts_.xml file from the Scaife title page. Go figure. I will fix all this. Thanks @lcerrato